docs/podman.1.md

```diff
@@ -182,13 +182,15 @@ the exit codes follow the `chroot` standard, see below:
 
 ## FILES
 
-**libpod.conf** (`/etc/containers/libpod.conf`)
+**libpod.conf** (`/usr/share/containers/libpod.conf`)
 
-    libpod.conf is the configuration file for all tools using libpod to manage containers.  When Podman runs in rootless mode, then the file `$HOME/.config/containers/libpod.conf` is used.
+    libpod.conf is the configuration file for all tools using libpod to manage containers, when run as root.  Administrators can override the defaults file by creating `/etc/containers/libpod.conf`.  When Podman runs in rootless mode, then the file `$HOME/.config/containers/libpod.conf` is created and overrides the system configuration file.
```

Comment:
Not completely - we supplement the homedir config file from the root one when we can (conmon and runc paths, for example)